qutip/tests/test_device.py

```diff
@@ -70,7 +69,7 @@
 
 
 device_lists = [
-    pytest.param(DispersiveCavityQED, {"g":0.1}, id = "DispersiveCavityQED"),
+    pytest.param(DispersiveCavityQED, {"g": 0.02}, id="DispersiveCavityQED"),
```

Comment:
What warning did `g = 0.1` trigger?

Reply:
It is a warning saying that `dispersive approximation may not be valid`, related to whether the physics model is accurate or not.

Reply:
Yeah, Boxi's right. The calls were wrapped in a `warnings.catch_warnings()` block inside the test, but I think that when you turn warnings into errors, pytest hooks itself in a little too deeply for that to work.